src/util/Constants.ts

```diff
@@ -0,0 +1 @@
+export { version } from "../../package.json"
```

Comment:
```suggestion
export { version } from "package.json"
```

Reply:
It was causing bugs at compile time so I excluded from changes.

Reply:
@wonderlandpark if you type anything here I’ll resolve this change request

Reply:
What bug

Reply:
The typescript compiler flag "--baseDir" was not working (but in vscode it seems working) successfully